### DISCUSSION TITLE
Dividers not moving to out of scope for out of scope filters

### DISCUSSION THREAD

**Discussion by superset-user** [apache/superset#25621] [Q&A / Help]
Hi all, we recently upgraded to superset 2.1.0. While assigning dividers to filters I see that divider names (in red) doesn't move to out of scope section for out of scope filters. Please find image below:
![Capture](https://github.com/apache/superset/assets/144122804/88253167-02f9-4eaf-b879-00d4fff5bf69)
As you can see out of scope filter are moving to out of scope section, but dividers (in red) belonging to these out of scope filters stay in above section. Is there any way to move these dividers to out of scope section along with filters? Thanks in advance!

**Comment by Marchena27** [comment]
I am also interested in this issue, does anyone know how to fix it or any alternative to split filter groups?

I have several tabs in my dashboard and I want the filters to appear next to the splitters when in the corresponding tab.
